documentation/business-analyst-activities.md

```diff
@@ -1 +1,17 @@
 # Business Analyst Activities
+
+**How much detail should we go into with regards to setting up GITHUB account?
+
+
+## Updating Polygon Mappings in the mapped_locations.yml file
+This section details the process for updating the search mappings so that a polygon search is used in place of a point search.
+
+1. Navigate to the "Code" tab within github and press "go to file"
+2. Search for "mapped_locations.yml" and click on the file.
```

Comment:
Could we put the link here instead/as well?

https://github.com/DFE-Digital/teaching-vacancies/blob/master/lib/tasks/data/mapped_locations.yml

Reply:
Have added link to doc.